### PR TITLE
ACS-6379: Fix docker-compose

### PR DESCRIPTION
**Pull request by krdabrowski**
- upgraded version to 3.8
- added healthcheck for alfresco container
- ordered container run: alfresco -> reindexer -> live-indexer

### CODE REVIEW

**Review by Pawel-608 — APPROVED** [review]
LGTM, it'll be really useful, thanks!

File: distribution/src/main/resources/docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -1,9 +1,12 @@
-version: "2"
+version: "3.8"
```
Comment (mpichura) [inline distribution/src/main/resources/docker-compose/docker-compose.yml:1]: I am not sure if that is compatible with GHA env setup - but assuming that this docker-compose is used in pipeline and tests are green.

**Review by mpichura — APPROVED** [review]
LGTM, have look at my comment.

File: distribution/src/main/resources/docker-compose/docker-compose.yml (hunk 1 above)
Comment (krdabrowski) [reply distribution/src/main/resources/docker-compose/docker-compose.yml:1]: This d-c is not used in pipeline - it's used only for local run of the app

**Review by tpage-alfresco — APPROVED** [review]
LGTM. Nice idea to include the reindexing component :+1:

**Pull request merged**